algorithms/src/std_algorithms/Kokkos_IterSwap.hpp

```diff
@@ -58,6 +57,16 @@ void iter_swap(IteratorType1 a, IteratorType2 b) {
   Impl::iter_swap_impl(a, b);
 }
 
+#ifdef KOKKOS_ENABLE_DEPRECATED_CODE_4
+template <class T>
+KOKKOS_DEPRECATED_WITH_COMMENT("Use Kokkos::kokkos_swap instead!")
+KOKKOS_FUNCTION
+    void swap(T& a, T& b) noexcept(::Kokkos::kokkos_swap(std::declval<T&>(),
+                                                         std::declval<T&>())) {
+  ::Kokkos::kokkos_swap(a, b);
+}
+#endif
+
```

Comment:
I'm guessing this in moved here just for the sake of deprecation warning and to delete `std_algorithms/Kokkos_Swap.hpp` right away?

Reply:
It is in order to by more time to anyone who would be using `Kokkos::Experimental::swap`